pkg/config_test.go

```diff
@@ -0,0 +1,64 @@
+package pkg
+
+import (
+	"io/ioutil"
+	"os"
+	"os/user"
+	"testing"
+)
+
+func TestGetregistryParams(t *testing.T) {
+	usr, err := user.Current()
+	if err != nil {
+		t.Logf("could not retrieve current user: %v", err)
+		t.SkipNow()
+	}
+	if usr.Uid != "0" {
+		t.Logf("current user does not have UID of zero (got %s) "+
+			"so cannot create registry cred location, skipping", usr.Uid)
+		t.SkipNow()
+	}
```

Comment:
the function i am trying to test already assumes some data is present in the `var/run/` directory and the only way i can test it is create some data in that directory but in most scenarios we won't have permissions to create data in that directory and hence it is not correct to assume it as a failure.
I will see if i can pass the directory as an argument so i can test it more effectively.